lib/fs.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const metasync = require('metasync');
+
+const error = (err, options, cb) => {
+  if (err.code === 'EEXIST') cb();
+  else if (!options.errorLog) cb(err);
+  else {
+    options.errorLog.write(JSON.stringify(err));
+    cb();
+  }
+};
+
+const cpDir = (src, dest, options, curDepth, cb) => {
+  if (curDepth > options.depth) {
+    cb();
+    return;
+  }
+  fs.mkdir(dest, err => {
+    if (err !== null && err.code !== 'EEXIST') {
+      error(err, options, cb);
+      return;
+    }
+    fs.readdir(src, (err, files) => {
+      if (err) {
+        error(err, options, cb);
+        return;
+      }
+      metasync.each(
+        files,
+        (file, cb) => {
+          if (
+            (options.filter instanceof RegExp && options.filter.test(file)) ||
+            (typeof options.filter === 'function' && options.filter(file))
+          ) {
+            cb();
+            return;
+          }
+          const srcFile = path.join(src, file);
+          fs.stat(srcFile, (err, stats) => {
+            if (err) {
+              error(err, options, cb);
+              return;
+            }
+            const srcFile = path.join(src, file);
+            const destFile = path.join(dest, file);
+            if (stats.isDirectory())
```

Comment:
```js
            if (stats.isDirectory()) {
              cpDir(srcFile, destFile, options, curDepth + 1, cb);
            } else if (stats.isFile()) {
```

Reply:
What is wrong? It is the same, or I just don't see something?

Reply:
Yes, formatting. If the code in `if` is too long to put it on one line or if there is `else` it should be wrapped in curly braces just like you did [here](https://github.com/metarhia/common/pull/251/files#diff-9a205ef7ee967ee32efee02e58b3482dR61). It is not obligatory, it doesn't impact performance, but it improves readability and makes easier to make changes in the future. You can check out code style recommendations [here](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/conditionals).